client/src/components/AboutDialog.vue

```diff
@@ -27,19 +27,23 @@
       <v-card-text
         class="px-4 pt-4 pb-1"
         >
-        Multinet is a system for storing and processing <a
-          href="https://vdl.sci.utah.edu/mvnv/" rel="noopener
-          noreferrer">multivariate networks</a>. Learn more and explore the code
-        at <a href="https://github.com/multinet-app/multinet" rel="noopener
-          noreferrer">GitHub</a>.
+        Multinet is a system for storing and processing <a-ext
+          href="https://vdl.sci.utah.edu/mvnv/">multivariate networks</a-ext>. Learn more and explore the code
+        at <a-ext href="https://github.com/multinet-app/multinet">GitHub</a-ext>.
+      </v-card-text>
+
+      <v-card-text class="px-4 pt-4 pb-1">
+        Check out the Multinet project <a-ext
+          href="https://multinet-app.readthedocs.io">documentation</a-ext>, or
+        the <a-ext href="/apidocs">API docs</a-ext>.
```

Comment:
Small thing, but maybe formatting like so would be more readable?
```
Check out the Multinet project 
<a-ext href="https://multinet-app.readthedocs.io">documentation</a-ext>
, or the 
<a-ext href="/apidocs">API docs</a-ext>.
```

Reply:
I just let my editor reflow the text in all such cases. I think doing it the way you suggest will introduce a space before the comma.